packages/opencensus-core/src/trace/model/span-base.ts

```diff
@@ -114,12 +114,12 @@ export abstract class SpanBase implements types.Span {
   }
 
   /** Gives the TraceContext of the span. */
-  get traceContext(): types.TraceContext {
+  get spanContext(): types.SpanContext {
```

Comment:
Why was the type/field changed to `SpanContext`? I thought `TraceContext` was fine. (If you decide to keep `SpanContext`, please change the code comment above.)

Actually, also looking at this more closely -- does the `parentSpanId` need to be there? It's not part of the `SpanContext` interface. Also, it would be great to have `options` here as well.

Reply:
>Why was the type/field changed to SpanContext? 
It was changed to be consistent with opencensus that uses the term SpanContext:
https://github.com/census-instrumentation/opencensus-java/blob/master/api/src/main/java/io/opencensus/trace/SpanContext.java

> Actually, also looking at this more closely -- does the parentSpanId need to be there? 
There is no parentSpanId in opencensus SpanContext interface. As I understand the spanId should represent a parendSpanID when the context is propagated.